Documentation/Includes.rst.txt

```diff
@@ -1,11 +1,12 @@
-﻿.. This is 'Includes.txt'. It is included at the very top of each and
+.. This is 'Includes.txt'. It is included at the very top of each and
    every ReST source file in THIS documentation project (= manual).
 
-.. role:: aspect (emphasis)
+.. role:: aspect(emphasis)
 .. role:: html(code)
 .. role:: js(code)
 .. role:: php(code)
-.. role:: sep (strong)
+.. role:: pn(emphasis)
```

Comment:
How about `name` instead?

Reply:
It will appear very often in text, therefore it's good to have a short role-name. IMHO four letters of 'name' would be ok, but 'name' has too many meanings already. For example many directives have an option 'name'. Editors will have to learn some rules anyway, and "proper nouns" is something mentioned there already. No matter of you think of "proper name" (= Eigenname, I looked it up) and "proper noun" would both fit to "pn". 
![grafik](https://user-images.githubusercontent.com/307057/102888622-71fb3a00-4459-11eb-8e33-97497400f1a2.png)

And it will be easy to grasp, even simply by looking at the text and the frontend rendering, if you have something like 'TYPO3 Core Team':pn:
And, I think it's good to have a slightly uncommon name. Makes it more unlikely to clash with what others in the world may come up with.

So I'm still +1 for 'pn'.

Reply:
I'm still not too fond of this non-speaking name but I respect your reasoning.